modules/apps/app-builder/app-builder-test/src/testFunctional/macros/FormViewBuilder.macro

```diff
@@ -191,6 +207,20 @@ definition {
 		AssertElementPresent(locator1 = "FormViewBuilder#OPTIONS_OPTION_NAME_FIELD");
 	}
 
+	macro editRules {
+		var key_ruleName = "${ruleName}";
+		var key_rowNumber = "${rowNumber}";
+		Click(locator1 = "FormViewBuilder#EDIT_RULE_COLLAPSE");
+		AssertElementPresent(locator1 = "FormViewBuilder#EDIT_RULE_EXPANDED");
+		Click(locator1 = "FormViewBuilder#EDIT_RULE");
+	}
+
+	macro editRulesDefault {
+		key_rowNumber = "${rowNumber}";
```

Comment:
Hi @calvinctvo , please remove the key_rowNumber since you don't need it anymore, once the selectors are updated.